sklearn/compose/_column_transformer.py

```diff
@@ -630,6 +631,10 @@ def _hstack(self, Xs):
             Xs = [f.toarray() if sparse.issparse(f) else f for f in Xs]
             return np.hstack(Xs)
 
+    def _sk_repr_html(self):
+        names, transformers, name_details = zip(*self.transformers)
+        return _EstHTMLBlock('parallel', transformers, names, name_details)
```

Comment:
maybe use `name_details=columns` to make the translation between the semantics here and the semantics there more explicit